artifactory/commands/docker/pull.go

```diff
@@ -24,10 +24,12 @@ func (dpc *DockerPullCommand) Run() error {
 	if errorutils.CheckError(err) != nil {
 		return err
 	}
-	loginConfig := &docker.DockerLoginConfig{ArtifactoryDetails: rtDetails}
-	err = docker.DockerLogin(imageTag, loginConfig)
-	if err != nil {
-		return err
+	if !dpc.skipLogin {
+		loginConfig := &docker.DockerLoginConfig{ArtifactoryDetails: rtDetails}
+		err = docker.DockerLogin(imageTag, loginConfig)
```

Comment:
I think we can remove the imageTag variable and just use the dpc.ImageTag() here. So the `	imageTag := dpc.ImageTag()` will be removed and instead as in the push command all will be done on the dpc.ImageTag()